README.md

```diff
@@ -331,4 +344,11 @@ The `outbound-edi` function can be invoked via the UI for testing.
 
   </details>
 
-5. You can view the file using the [Buckets Web View](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
\ No newline at end of file
+5. You can view the file using the [Buckets Web View](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
```

Comment:
```suggestion
5. You can view the file using the [Buckets UI](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
```

Reply:
I know you didn’t change this, Laura, but I missed this before. This is another thing we haven’t been able to agree on, but the docs are pretty consistent these days in using _UI_ for the, well, UI.

Reply:
thank you!! I am always a fan of opportunistic fixes :)